repair/repair_task.hh

```diff
@@ -259,6 +278,10 @@ public:
         , _ri(ri)
         , _ex(ex)
     {}
+
+    future<task_manager::task::progress> gather_child_progress(unsigned i) const override {
```

Comment:
should be virtual override

Reply:
that said, submit_to the same shard costs very little, so I'm not sure virtualizing this method makes sense.
It doesn't save much in this case.
Do we have a need for a radically different way of getting the child progress?